dags/bqetl_core.py

```diff
@@ -0,0 +1,59 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "jklukas@mozilla.com",
+    "start_date": datetime.datetime(2019, 7, 25, 0, 0),
+    "email": ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=300),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+}
+
+with DAG("bqetl_core", default_args=default_args, schedule_interval="0 1 * * *") as dag:
+
+    telemetry_derived__core_clients_daily__v1 = bigquery_etl_query(
+        task_id="telemetry_derived__core_clients_daily__v1",
+        destination_table="core_clients_daily_v1",
+        dataset_id="telemetry_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="jklukas@mozilla.com",
+        email=["jklukas@mozilla.com"],
```

Comment:
Does `email` get automatically populated based on `owner` in the `metadata.yaml` for the query? It seems we should respect the `email` set in `default_args` instead. Or perhaps merge owner in with the `email` set in default_args. Probably out of scope here.

Reply:
Currently, `email` for tasks is a set of the owner email plus the emails set in the tasks `scheduling` config. We could think of changing that and use the emails set in the DAG config instead.